integrationTests/oneNodeNetwork.go

```diff
@@ -0,0 +1,99 @@
+package integrationTests
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+type oneNodeNetwork struct {
+	advertiser   p2p.Messenger
+	Nodes        []*TestProcessorNode
```

Comment:
why multiple nodes inside in an oneNodeNetwork?

Reply:
done